index.html

```diff
@@ -484,6 +490,115 @@
               layout="[[layoutClass]]"
               route="{{subRoute}}"></generic-page>
           </template>
+          <!-- LURES    ============================================   -->
```

Comment:
@ebrent8 Is there a UX design for how to users are intended to navigate through the lures on the site?

Lures represent a new content type and should probably not be at the top level; if there is an information architecture / interaction design, I would be happy to template out the implementation solution to support the interaction.

In general, the top level pages are all (from an URL standpoint) board agnostic, intended to take a parameter that would then provide board specific details. There were a couple special cases to that--`max-hse` for example--which were implemented as small special cases.

Once a top-level page lands, and starts being linked to, the top level URL navigation becomes fairly locked. We don't want everything to end up being on the top-level.